setup.py

```diff
@@ -13,7 +13,14 @@
       author_email='archive@rockarch.org',
       license='MIT',
       packages=find_packages(),
-      install_requires=[],
+      install_requires=["ArchivesSnake>=0.8.1",
+                        "boltons>=20.0.0",
+                        "fuzzywuzzy>=0.17.0",
+                        "pre-commit>=1.18.3"
+                        "pytest",
+                        "PyYAML>=5.2.0",
+                        "sphinx>=1.8.5",
+                        "tox>=3.14.0"],
```

Comment:
```suggestion
      install_requires=["ArchivesSnake>=0.8.1",
                        "fuzzywuzzy>=0.17.0"],
```
IIRC, the install really only requires these two. The others are a) dependencies of ASnake or b) used in testing.

Reply:
It's also possible to not pin versions, and just let pip install the most recent version of the package. Something to think about.

Reply:
@helrond Yeah, I saw that you can just include the package, but what I was reading is that it's best practice to include any known lower or upper bounds for your project, so I'd argue for keeping them in.

Reply:
That sounds reasonable to me!